packages/garbo/src/familiar/lib.ts

```diff
@@ -131,3 +134,22 @@ export function estimatedBarfExperience(): number {
 
   return sumNumbers(sources);
 }
+
+const ESTIMATED_ENVELOPE_VALUE = 50_000;
+
+export function snapperValue(): number {
+  const item = Snapper.phylumItem.get(globalOptions.target.phylum);
+  if (!item) return 0;
+  
+  const denominator =
+    11 -
+    (Snapper.getTrackedPhylum() === globalOptions.target.phylum
+      ? Snapper.getProgress()
+      : 0);
+  if (denominator > copyTargetCount()) return 0;
+
+  if (item === $item`envelope full of Meat`) {
```

Comment:
you can handle this in [value.ts](https://github.com/loathers/garbage-collector/blob/f22fa29b29e2f2897906e2e2392cb9a84811c14d/packages/garbo-lib/src/value.ts#L32)